src/server/pfs/server/driver.go

```diff
@@ -219,7 +219,7 @@ func (d *driver) createRepo(txnCtx *txncontext.TransactionContext, repo *pfs.Rep
 				txnCtx,
 				whoAmI.Username,
 				[]string{auth.RepoOwnerRole},
-				&auth.Resource{Type: auth.ResourceType_REPO, Name: repo.Name},
+				&auth.Resource{Type: auth.ResourceType_REPO, Name: pfsdb.RepoKey(repo)},
```

Comment:
Should this be the database key of the repo? That feels like using package `pfsdb` for something unrelated to the PFS database layer. Maybe this should be the result of `repo.String` instead?